BlueprintUICommonControls.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
 
   s.swift_version = '5.1'
 
-  s.ios.deployment_target = '12.0'
+  s.ios.deployment_target = '11.0'
```

Comment:
Agh how did I miss this one

Reply:
Oh wait this is just out of date

Reply:
LOL I reverted some stuff so I can test in register